apps/mobile/src/queries/market-data/stx-market-data.query.ts

```diff
@@ -1,22 +1,8 @@
-import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
+import { stxCryptoAsset } from '@leather.io/constants';
 
-import { getMarketDataService } from '@leather.io/services';
-import { oneMinInMs } from '@leather.io/utils';
-
-export function createStxMarketDataQueryOptions() {
-  return {
-    queryKey: ['market-data-service-get-stx-market-data'],
-    queryFn: ({ signal }: QueryFunctionContext) => getMarketDataService().getStxMarketData(signal),
-    refetchOnReconnect: false,
-    refetchOnWindowFocus: false,
-    refetchOnMount: false,
-    retryOnMount: false,
-    staleTime: oneMinInMs,
-    gcTime: oneMinInMs,
-  } as const;
-}
+import { useMarketDataQuery } from './market-data.query';
 
 // LEA-1761 FIXME: this query returns the wrong precision - price in dollars rather than cents
 export function useStxMarketDataQuery() {
-  return useQuery(createStxMarketDataQueryOptions());
+  return useMarketDataQuery(stxCryptoAsset);
```

Comment:
I think we can remove the `// LEA-1761 FIXME: th` now as it was solved elsewhere